sprokit/processes/adapters/tests/test_non_blocking.cxx

```diff
@@ -120,7 +120,7 @@ main(int argc, char* argv[])
 }
 
 // ==================================================================
-IMPLEMENT_TEST(non_blocking)
+IMPLEMENT_TEST(nonblocking)
```

Comment:
Does this expand to "test_non_blocking"? Otherwise it's not obvious where this conflicts? It seems unfortunate to make this name different from the test itself.